### DISCUSSION TITLE
Welcome to Duino-Coin Discussions!

### DISCUSSION THREAD

**Discussion by revoxhere** [revoxhere/duino-coin#294] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by IdotMaster1** [comment]


**Reply by ghost** [reply]
h

**Reply by ahmedhussien42** [reply]
hi there
when i try to use the ESP32 dual core code i noticed that the esp32 hashrate decay after almost 5 min. to become less than 1k and as per the code it disconnect from the server and reconnect twice times then the esp32 hangs - is there any solution for that

**Reply by revoxhere** [reply]
@JoyBed will be able to help

**Reply by ahmedhussien42** [reply]
okay, waiting for his reply

**Comment by ahmedhussien42** [comment]
![Untitled](https://user-images.githubusercontent.com/81731924/113244944-45f02000-92b6-11eb-9827-8edb5481fc7a.png)

**Reply by ahmedhussien42** [reply]
![image](https://user-images.githubusercontent.com/81731924/113245032-6f10b080-92b6-11eb-801f-451207fd8cfc.png)

**Comment by ahmedhussien42** [comment]
![image](https://user-images.githubusercontent.com/81731924/113245612-99af3900-92b7-11eb-9321-240183f492dd.png)

**Comment by ahmedhussien42** [comment]
Greetings

How to contact him

regards

Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows 10

**Comment by tarbear123** [comment]
Will i get my account suspended if I mine on several devices but all connected to the same account?

**Reply by revoxhere** [reply]
No, as long as you use less than 24 devices for mining.

**Reply by tarbear123** [reply]
ok thanks

**Comment by pappazzo** [comment]
Hi, I'm trying to register a wallet account via web. But once I click on the register button, nothing happens.
Can you help me?

**Reply by revoxhere** [reply]
Try again now.

**Comment by shaneorourke** [comment]
When I run CLI_Wallet.exe on windows I get the install tronpy error. Which I've now installed (along with some C++ prerequisites) but I still get the error.
Even when I install tronpy via python pip it says "already satisfied" meaning that it's installed. I still get the error.

P.S. This only affects the wrapper functionality. Also errors on Linux and windows Wallet.exe

**Comment by eb1991** [comment]
Is there a way to connect Arduino nano clone to node MCU board instead of my loud PC? I mean is someone tried to write code for esp8266 that works like avr_miner. Right now I am mining with 3 esp8266 (2 nodemcu's and 1 wemos) since I first tried out esp my last Arduino is just collecting dust so it could be nice to think that it is doing something. Problem is that I am home and awake only 4 hours a day and it would be kind of stupid to leave PC on 24/7

**Comment by eb1991** [comment]
Tried out PC_Miner.py on Linux mint works fine (got 3.5 - 4 MH/s) but on my windows laptop PC_Miner.exe not working v2.2 works but don't mine because of old version. 2.4.5 don't open terminal it says can't reach file or not allowed. Tried compatibility mode and run as administrator with no result

**Comment by boblach** [comment]
how go you know if browser is  working?
